far/FarEng.hlf.m4

```diff
@@ -6121,43 +6121,56 @@ files, like #Temporary panel# does.
 
 @System.CopySecurityOptions
```

Comment:
Original English translation for reference:
https://github.com/FarGroup/FarManager/blob/release/2.0/docs/ENG/TechInfo.txt#L539-L580

Reply:
Thanks, I've seen it. It is difficult to understand. I liberally reworked both English and Russian versions.